packages/holocron/__tests__/loadModule.node.spec.js

```diff
@@ -422,17 +439,59 @@ describe('loadModule.node', () => {
     expect(requireFromString).toHaveBeenCalledWith('hello there!', 'https://example.com/cdn/awesome/1.0.0/awesome.node.js');
   });
 
-  it('does not validate integrity if NODE_ENV is development', async () => {
-    expect.assertions(3);
-    const integrity = 'invalid sha!!!';
-    process.env.NODE_ENV = 'development';
+  describe('in development ', () => {
+    beforeEach(() => {
+      process.env.NODE_ENV = 'development';
+    });
 
-    const loadModule = load({ fetchStatus: 200, fetchText: 'hello there!' });
-    moduleRegistry.addToModuleBlockList = jest.fn();
-    await expect(
-      loadModule('awesome', { node: { integrity, url: 'https://example.com/cdn/awesome/1.0.0/awesome.node.js' } })
-    ).resolves.toEqual({ str: 'hello there!' });
-    expect(moduleRegistry.addToModuleBlockList).not.toHaveBeenCalled();
-    expect(requireFromString).toHaveBeenCalledWith('hello there!', 'https://example.com/cdn/awesome/1.0.0/awesome.node.js');
+    const moduleName = 'broken-dev-module';
+    const moduleUrl = `https://example.com/cdn/${moduleName}/1.0.0/${moduleName}.node.js`;
+
+    it('does not validate integrity if NODE_ENV is development', async () => {
+      expect.assertions(3);
+      const integrity = 'invalid sha!!!';
+
+      const loadModule = load({ fetchStatus: 200, fetchText: 'hello there!' });
+      moduleRegistry.addToModuleBlockList = jest.fn();
+      await expect(
+        loadModule('awesome', { node: { integrity, url: 'https://example.com/cdn/awesome/1.0.0/awesome.node.js' } })
+      ).resolves.toEqual({ str: 'hello there!' });
+      expect(moduleRegistry.addToModuleBlockList).not.toHaveBeenCalled();
+      expect(requireFromString).toHaveBeenCalledWith('hello there!', 'https://example.com/cdn/awesome/1.0.0/awesome.node.js');
+    });
+
+    it(`does not add \`${moduleName}\` to block list in development`, async () => {
+      expect.assertions(4);
+
+      const requireError = new Error(`err... ${moduleName} failed to load`);
+      requireError.shouldBlockModuleReload = false;
+      const loadModule = load();
+      moduleRegistry.addToModuleBlockList = jest.fn();
+      requireFromString.mockImplementation(() => { throw requireError; });
+
+      await expect(
+        loadModule(moduleName, { node: { url: moduleUrl } })
+      ).rejects.toBe(requireError);
+      expect(fetch).toHaveBeenCalledTimes(1);
+      expect(console.warn).toHaveBeenCalledTimes(1);
+      expect(moduleRegistry.addToModuleBlockList).not.toHaveBeenCalledWith(moduleUrl);
+    });
+
+    it(`adds \`${moduleName}\` to block list if \`shouldBlockModuleReload\` is 'true' and in development`, async () => {
```

Comment:
module should never get added to blocklist in development

Reply:
I've updated 5233682